{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/urls.py

```diff
@@ -1,6 +1,9 @@
 from django.contrib import admin
 from django.urls import include, path
 
+admin.site.site_header = "{{ cookiecutter.project_name }} Admin"
+admin.site.site_title = "{{ cookiecutter.project_name }}"
+
```

Comment:
Just a nice little addition so that managing multiple projects makes it more obvious which admin you are in.

Also, non-dev project stakeholders don't know (or need to care) what the "Django Admin" is. It's just the admin for the app.